native/cocos/editor-support/spine-wasm/AtlasAttachmentLoaderExtension.cpp

```diff
@@ -40,9 +40,10 @@ void AtlasAttachmentLoaderExtension::configureAttachment(Attachment *attachment)
         auto *region = static_cast<AtlasRegion *>(regionAttachment->getRendererObject());
         auto *attachmentVertices = new AttachmentVertices(4, quadTriangles, 6, pages.indexOf(region->page));
         V3F_T2F_C4B *vertices = attachmentVertices->_triangles->verts;
+        const auto &uvs = regionAttachment->getUVs();
         for (int i = 0, ii = 0; i < 4; ++i, ii += 2) {
-            vertices[i].texCoord.u = regionAttachment->getUVs()[ii];
-            vertices[i].texCoord.v = regionAttachment->getUVs()[ii + 1];
+            vertices[i].texCoord.u = uvs[ii];
+            vertices[i].texCoord.v = uvs[ii + 1];
```

Comment:
Avoid invoking `getUVs` functions many times which will save some code size and improve performance a little bit.